src/box/memtx_bitset.cc

```diff
@@ -498,38 +499,41 @@ memtx_bitset_index_count(struct index *base, enum iterator_type type,
 }
 
-static const struct index_vtab memtx_bitset_index_vtab = {
-	/* .destroy = */ memtx_bitset_index_destroy,
-	/* .commit_create = */ generic_index_commit_create,
-	/* .abort_create = */ generic_index_abort_create,
-	/* .commit_modify = */ generic_index_commit_modify,
-	/* .commit_drop = */ generic_index_commit_drop,
-	/* .update_def = */ generic_index_update_def,
-	/* .depends_on_pk = */ generic_index_depends_on_pk,
-	/* .def_change_requires_rebuild = */
+static const struct memtx_index_vtab memtx_bitset_index_vtab = {
+	/* .index_vtab = */
+	{
+		/* .destroy = */ memtx_bitset_index_destroy,
+		/* .commit_create = */ generic_index_commit_create,
+		/* .abort_create = */ generic_index_abort_create,
+		/* .commit_modify = */ generic_index_commit_modify,
+		/* .commit_drop = */ generic_index_commit_drop,
+		/* .update_def = */ generic_index_update_def,
+		/* .depends_on_pk = */ generic_index_depends_on_pk,
+		/* .def_change_requires_rebuild = */
 		memtx_index_def_change_requires_rebuild,
-	/* .size = */ memtx_bitset_index_size,
-	/* .bsize = */ memtx_bitset_index_bsize,
-	/* .quantile = */ generic_index_quantile,
-	/* .min = */ generic_index_min,
-	/* .max = */ generic_index_max,
-	/* .random = */ generic_index_random,
-	/* .count = */ memtx_bitset_index_count,
-	/* .get_internal = */ generic_index_get_internal,
-	/* .get = */ generic_index_get,
+		/* .size = */ memtx_bitset_index_size,
+		/* .bsize = */ memtx_bitset_index_bsize,
+		/* .quantile = */ generic_index_quantile,
+		/* .min = */ generic_index_min,
+		/* .max = */ generic_index_max,
+		/* .random = */ generic_index_random,
+		/* .count = */ memtx_bitset_index_count,
+		/* .get_internal = */ generic_index_get_internal,
+		/* .get = */ generic_index_get,
+		/* .create_iterator = */ memtx_bitset_index_create_iterator,
+		/* .create_iterator_with_offset = */
+		generic_index_create_iterator_with_offset,
+		/* .create_arrow_stream = */ generic_index_create_arrow_stream,
+		/* .create_read_view = */ generic_index_create_read_view,
+		/* .stat = */ generic_index_stat,
+		/* .compact = */ generic_index_compact,
+		/* .reset_stat = */ generic_index_reset_stat,
+	},
 	/* .replace = */ memtx_bitset_index_replace,
-	/* .create_iterator = */ memtx_bitset_index_create_iterator,
-	/* .create_iterator_with_offset = */
-	generic_index_create_iterator_with_offset,
-	/* .create_arrow_stream = */ generic_index_create_arrow_stream,
-	/* .create_read_view = */ generic_index_create_read_view,
-	/* .stat = */ generic_index_stat,
-	/* .compact = */ generic_index_compact,
-	/* .reset_stat = */ generic_index_reset_stat,
-	/* .begin_build = */ generic_index_begin_build,
-	/* .reserve = */ generic_index_reserve,
-	/* .build_next = */ generic_index_build_next,
-	/* .end_build = */ generic_index_end_build,
+	/* .begin_build = */ generic_memtx_index_begin_build,
+	/* .reserve = */ generic_memtx_index_reserve,
+	/* .build_next = */ generic_memtx_index_build_next,
+	/* .end_build = */ generic_memtx_index_end_build,
```

Comment:
I'd organized a bit differently:

```c
static const struct index_vtab memtx_bitset_index_vtab_base = {
...
};

static const struct memtx_index_vtab memtx_bitset_index_vtab = {
        /* .base = */ memtx_bitset_index_vtab_base,
...
};
```

Keeps basic index structe, less indentation, and less diff also.